### PR TITLE
Add FAQ item for cases when ECID is cleared or reset

### PR DESCRIPTION
**Pull request by timkimadobe**
## Description

This PR adds an FAQ item for Identity for Edge Network FAQ page to cover cases when the ECID is reset or cleared.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by calebk1 — APPROVED** [review]
LGTM!

File: src/pages/edge/identity-for-edge-network/faq.md [hunk 1]
```diff
@@ -113,6 +113,16 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The ECID is reset or cleared in the following cases:
+* Cleared when the app is uninstalled.
```
Comment (calebk1) [inline src/pages/edge/identity-for-edge-network/faq.md:119]: I think to make this not fail validation, there needs to be a newline between here and the beginning of the unordered list.

File: src/pages/edge/identity-for-edge-network/faq.md [hunk 2]
```diff
@@ -113,6 +113,17 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The ECID is reset or cleared in the following cases:
+
+* Cleared when the app is uninstalled.
+* Reset using the [`resetIdentities`](../../home/base/mobile-core/api-reference.md#resetidentities) API.
+* Reset when the app's local persistence cleared (by the app developer, other SDKs, etc.).
```
Comment (emdobrin) [inline src/pages/edge/identity-for-edge-network/faq.md:122]: Rephrasing suggestion:
Reset in the case of app local persistence is cleared for any reason in the app implementation. To learn more, see where the Mobile SDK store identities data. (relative link to https://developer.adobe.com/client-sdks/resources/faq/#where-does-the-sdk-store-identities-and-preferences-on-the-app) 

And I think we can remove the details about persistence.

File: src/pages/edge/identity-for-edge-network/faq.md [hunk 3]
```diff
@@ -113,6 +113,17 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The ECID is reset or cleared in the following cases:
```
Comment (emdobrin) [inline src/pages/edge/identity-for-edge-network/faq.md:118]: Can we add a clarifying comment: Identity for Edge Network extension does not reset or clear the ECID. If you are seeing the ECID being reset, identify if it is one of the following cases:

File: src/pages/edge/identity-for-edge-network/faq.md [hunk 4]
```diff
@@ -113,6 +113,14 @@ To clear the identifiers used by the Adobe Solutions extensions, call [setPrivac
 
 To clear the identifiers used by the Edge extensions, call [resetIdentities](../../home/base/mobile-core/api-reference.md#resetidentities)
 
+## Q: In what cases is the Experience Cloud ID (ECID) reset or cleared?
+
+A: The Identity for the Edge Network extension does not automatically reset or clear the ECID. If you observe the ECID being reset, identify if it falls into one of the following cases:
+
+* Cleared when the app is uninstalled.
+* Reset using the [`resetIdentities`](../../home/base/mobile-core/api-reference.md#resetidentities) API.
+* Reset when the app local persistence is cleared for any reason in the app implementation. To learn more, see where the [Mobile SDK store identities data](../../../pages/resources/faq.md#where-does-the-sdk-store-identities-and-preferences-on-the-app).
```
Comment (kevinlind) [inline src/pages/edge/identity-for-edge-network/faq.md:122]: Suggest reword: "see where the Mobile SDK stores identity data"

**Review by calebk1 — APPROVED** [review]
LGTM!

File: src/pages/edge/identity-for-edge-network/faq.md (hunk 2 above)
Comment (emdobrin) [reply src/pages/edge/identity-for-edge-network/faq.md]: yes, that looks good to me - cc @praveek

**Review by emdobrin — APPROVED** [review]

**Pull request merged**